pgrx/src/lib.rs

```diff
@@ -97,7 +96,7 @@ pub use htup::*;
 pub use inoutfuncs::*;
 pub use itemptr::*;
 #[cfg(feature = "cshim")]
-pub use list::*;
+pub use list::old_list::*;
```

Comment:
I’m reading too-down so apologies if it’s settled elsewhere… old_list should be behind the cshim feature.